### PR TITLE
EDU-13141 - Category cover: Working with Site Editor

### PR DESCRIPTION
**Pull request by barbara-celi**
#### Types of changes
- [X] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

[EDU-13141
](https://vtex-dev.atlassian.net/browse/EDU-13141)

### CODE REVIEW

**Comment by barbara-celi** [comment]
sidebar + redirect: PR [#758](https://github.com/vtexdocs/devportal/pull/758)

File: docs/guides/vtex-io/Storefront-Guides/working-with-site-editor.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+slug: "working-with-site-editor"
+hidden: false
+createdAt: "2024-09-06T14:53:36.032Z"
+updatedAt: ""
+excerpt: "Discover how Site Editor simplifies the customization of your Store Framework storefront."
+---
+
+Site Editor works as a Content Management System (CMS) designed for stores developed with [Store Framework](https://developers.vtex.com/docs/guides/vtex-io-documentation-what-is-vtex-store-framework), reflecting the blocks defined in the [Store Theme](https://developers.vtex.com/docs/guides/vtex-io-documentation-store-theme) of the VTEX account.
+
+Based on the source code, the Site Editor section on the VTEX Admin displays an editable version of the website's storefront, enabling non-technical users to manage content as desired and modifying components and their behavior through a friendly interface.
```
Comment (carolinamenezes) [inline docs/guides/vtex-io/Storefront-Guides/working-with-site-editor.md:12]:
```suggestion
Based on the source code, the Site Editor displays an editable version of the website's storefront, enabling non-technical users to manage content as desired and modifying components and their behavior through a friendly interface.
```

File: docs/guides/vtex-io/Storefront-Guides/working-with-site-editor.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+---
+title: "Working with Site Editor"
+slug: "working-with-site-editor"
+hidden: false
+createdAt: "2024-09-06T14:53:36.032Z"
+updatedAt: ""
+excerpt: "Discover how Site Editor simplifies the customization of your Store Framework storefront."
+---
+
+Site Editor works as a Content Management System (CMS) designed for stores developed with [Store Framework](https://developers.vtex.com/docs/guides/vtex-io-documentation-what-is-vtex-store-framework), reflecting the blocks defined in the [Store Theme](https://developers.vtex.com/docs/guides/vtex-io-documentation-store-theme) of the VTEX account.
```
Comment (carolinamenezes) [inline docs/guides/vtex-io/Storefront-Guides/working-with-site-editor.md:10]:
```suggestion
[Site Editor](https://help.vtex.com/tutorial/site-editor-overview) works as a Content Management System (CMS) designed for stores developed with [Store Framework](https://developers.vtex.com/docs/guides/vtex-io-documentation-what-is-vtex-store-framework), reflecting the blocks defined in the [Store Theme](https://developers.vtex.com/docs/guides/vtex-io-documentation-store-theme) of the VTEX account.
```

File: docs/guides/vtex-io/Storefront-Guides/working-with-site-editor.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+Based on the source code, the Site Editor section on the VTEX Admin displays an editable version of the website's storefront, enabling non-technical users to manage content as desired and modifying components and their behavior through a friendly interface.
+
+In your VTEX Admin, go to **Storefront > Site Editor** to see the main available tools.
+
+![site-editor](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/vtex-io-documentation-site-editor-0.png)
+
+Changes made in Site Editor are immediately reflected in the store, and every storefront block can have its properties changed there. For example, you can change the content of a rich text block and shelf properties, such as which collection is being shown, among other changes.
+
+In this context, developers are responsible for building custom components tailored to specific needs, defining the structure and behavior of these components within the Store Theme, and ensuring seamless integration with Site Editor. This integration allows non-technical users to manage and update components easily, empowering store administrators to make changes to their storefronts without modifying the codebase.
```
Comment (carolinamenezes) [inline docs/guides/vtex-io/Storefront-Guides/working-with-site-editor.md:20]:
```suggestion
> ℹ️ Developers are responsible for creating components, defining the structure of the [Store Theme](https://developers.vtex.com/docs/guides/vtex-io-documentation-store-theme), and ensuring seamless integration with Site Editor. This integration allows non-technical users to manage and update components easily, empowering store administrators to make changes to their storefronts without modifying the codebase.
```

File: docs/guides/vtex-io/Storefront-Guides/working-with-site-editor.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+Learn more in [Site Editor - Overview](https://help.vtex.com/tutorial/site-editor-overview--299Dbeb9mFczUTyNQ9xPe1).
+
+## Before you begin
+
+<Steps>
+
+### Develop your Store Theme
+
+Make sure your store has a Store Theme developed following the [Storefront](https://developers.vtex.com/docs/guides/getting-started-3) guide.
```
Comment (carolinamenezes) [inline docs/guides/vtex-io/Storefront-Guides/working-with-site-editor.md:30]:
```suggestion
Make sure your store has a [Store Theme](https://developers.vtex.com/docs/guides/vtex-io-documentation-store-theme) developed following the [Storefront](https://developers.vtex.com/docs/guides/getting-started-3) guide.
```

**Review by carolinamenezes — APPROVED** [review]
😍 🎉

**Pull request merged**